usr/local/bin/install-um-extras

```diff
@@ -1,15 +1,119 @@
 #!/bin/bash
+# Install packages for the UM and UMDPs.
+# Allows a choice of GRIB_API and EcCodes, which are mutually exclusive.
 set -eu
 
+ubuntu_major=$(lsb_release -rs | cut -d. -f1)
+
+function usage {
+  # Print command line options
+  echo
+  echo 'Usage: um-setup [-e|--eccodes] [-g|--grib_api] [-v|--version <x.y>]'
+  echo
+  echo 'Options:'
+  echo '  -e, --eccodes        Install the EcCodes GRIB library'
+  echo '  -g, --grib           Install the GRIB_API GRIB library'
+  echo '  -h, --help           Show this help and exit'
+  echo '  -v, --version <x.y>  Install packages for the selected UM version'
+  echo
+  echo 'Provide at most one argument.'
+  echo 'Recommended usage is to indicate which UM version you wish to work with, e.g.'
+  echo '  um-setup -v 11.2'
+  echo 'For UM 11.2 or later, the EcCodes library for GRIB will be installed.'
+  echo 'For UM 11.1 or earlier, the GRIB_API library for GRIB will be installed.'
+  echo 'Alternatively, the --eccodes or --grib_api arguments may be provided to'
+  echo 'explicitly choose one library over another.'
+  echo 'These two libraries are mutually exclusive; only one may be installed ata time.'
```

Comment:
`ata` -> `at a`

Reply:
Well spotted. I've also clarified the help text as it wasn't clear that the script would install other things as well, and fixed the name of the grib-api option.